ext/standard/basic_functions.stub.php

```diff
@@ -2050,7 +2050,7 @@ function set_include_path(string $include_path): string|false {}
 function get_include_path(): string|false {}
 
 /** @refcount 1 */
-function print_r(mixed $value, bool $return = false): string|bool {}
+function print_r(mixed $value, bool $return = false): string|false {}
```

Comment:
This change is incorrect and contradictory to the upgrading entry. It's supposed to be `string|true`.